### DISCUSSION TITLE
Welcome to KaTeX Discussions!

### DISCUSSION THREAD

**Discussion by ylemkimon** [KaTeX/KaTeX#2404] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by k4b7** [comment]
Thanks for enabling this.  I've posted a comment about this in https://gitter.im/KaTeX/KaTeX.

**Comment by thomasgrafikus** [comment]


**Reply by edemaine** [reply]
Hi @thomasgrafikus, if you have a question, you should go to the [general discussion page](https://github.com/KaTeX/KaTeX/discussions) (not this particular thread) and ask it there. We'd need more information to help you (what are you trying to do? how did you try? what's going wrong?), so be sure to include that as well.

**Reply by thomasgrafikus** [reply]
Hi, thank you. I realized y ended up in the wrong place, I was searching support for the Katex Wordpress plugin, I’ sorry. I will delete this.
